### DISCUSSION TITLE
I don't see options for "Personal Finance" in the app

### DISCUSSION THREAD

**Discussion by asimthapa** [TreyWW/MyFinances#157] [Q&A] [answered]
As per the description of MyFinances, you can track personal finances. But I don't see any options for personal finance (invoices and receipts have specific uses, but personal expenses would be something simple as income, expense, investment or something like that). Did I not build the project properly ? I would like to know more about the goal/vision of this project because currently it doesn't seem to have any feature for "personal" finance. Maybe the feature is just not implemented currently ?
Also to make it clear, I'm not trying to complain, just trying to understand what the project is about.

<img width="1236" alt="Screenshot 2023-12-15 at 1 11 22 PM" src="https://github.com/TreyWW/MyFinances/assets/46013321/97c4674a-8422-4511-b9c1-61a1e09c5b84">

**Comment by TreyWW** [comment, marked as answer]
Hi @asimthapa,

Great question and this is a clear example of where I've kind of provided a false sense of progress that we've made.
At the moment the ability to log purchases, set goals, and swap between a Business/Individual account are all not yet possible unfortunately. They are the next features to come out though!

We are indeed lacking the good, needed features. The build you made seems to be up to date, glad you managed to install it fine.

Any of the features advertised on the home page that you can't see on the dashboard probably are awaiting development (can find some of them in https://github.com/TreyWW/MyFinances/issues - but if you  can't see one thats there feel free to create an issue!

Sorry for the inconvenience, it's my fault that the README has so much false advertisement.

Thanks for creating the discussion though, happy to answer any more questions regarding this topic or anything else!

**Reply by asimthapa** [reply]
oh nice, on the bright side I get to contribute more then ! Couple of things:

1. Some of the links in the new documentation is broken (getting-setup/other-environments/dependencies, getting-setup/other-environments/database
3. We can make the documentation more clear. For example, there is no explanation on what requirement.txt to use depending on what db to use (requirements.txt for mysql). Also I have to add database_type manually to .env file but it's not mentioned anywhere), etc.
4. Is  there any social group (discord,etc) for the project ?


Happy to contribute to the "Get Setup with the project" part of documentation, since I setup the project recently, so I have some idea on the problems/issues/confusion I faced as a new contributor.

**Reply by TreyWW** [reply]
That's amazing feedback, thank you! 

Great timing for documentation, i'm possibly considering moving to MkDocs rather than Docsify, so would be quite good to fix them issues along the way.

We have do have a [discord server](https://discord.gg/P9jJQacu6S).

Great feedback for the getting started guide, definitely would be good to add database_type to .env.sample, and improve docs further for starter users.

As for requirements.txt, i'm also setting up Poetry at the moment which hopefully may clear up some dependencies like Mysql, but may need to be outlined in the docs too!
